docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -170,20 +128,16 @@ WARNING: Session Status iframe functionality is limited in some modern browsers.
 [[_javascript_implicit_flow]]
 ==== Implicit and hybrid flow
 
-By default, the JavaScript adapter uses the https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code] flow.
+By default, the adapter uses the https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authorization Code] flow.
 
-With this flow the {project_name} server returns an authorization code, not an authentication token, to the application. The JavaScript adapter exchanges
-the `code` for an access token and a refresh token after the browser is redirected back to the application.
+With this flow the {project_name} server returns an authorization code, not an authentication token, to the application. The JavaScript adapter exchanges the `code` for an access token and a refresh token after the browser is redirected back to the application.
 
-{project_name} also supports the https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth[Implicit] flow where an access token
-is sent immediately after successful authentication with {project_name}. This may have better performance than standard flow, as there is no additional
-request to exchange the code for tokens, but it has implications when the access token expires.
+{project_name} also supports the https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth[Implicit] flow where an access token is sent immediately after successful authentication with {project_name}. This may have better performance than standard flow, as there is no additional request to exchange the code for tokens, but it has implications when the access token expires.
```

Comment:
```suggestion
{project_name} also supports the https://openid.net/specs/openid-connect-core-1_0.html#ImplicitFlowAuth[Implicit] flow where an access token is sent immediately after successful authentication with {project_name}. This flow may have better performance than standard flow because no additional request exists to exchange the code for tokens, but it has implications when the access token expires.
```